doc/nmstatectl.8.in

```diff
@@ -12,6 +12,8 @@ nmstatectl \- A nmstate command line tool
 .B nmstatectl rollback \fR[\fICHECKPOINT_PATH\fR]
 .br
 .B nmstatectl commit \fR[\fICHECKPOINT_PATH\fR]
+.br
+.B nmstatectl version
```

Comment:
I probably missed the change of introducing `version` as a command.
It makes no sense to me and I do not remember any command line tool having it as well.

But I guess this PR only reflects what exists, so by itself this is fine.

Reply:
Just for the records, some other tools that support a `version` command are: git, hg, oc, restic,